x-pack/legacy/plugins/siem/server/lib/events/query.events_over_time.dsl.ts

```diff
@@ -21,6 +21,7 @@ export const buildEventsOverTimeQuery = ({
         [timestamp]: {
           gte: from,
           lte: to,
+          ...(timezone && { time_zone: timezone }),
```

Comment:
I had to rename to `timezone` in the argument and reassign to `time_zone` because TS was angry about it not being camelcase once I do a conditional check on it. However, I still wanted it to be `time_zone` in the `TimerangeInput` type as this is the name of the Elasticsearch key for this field and the other fields have been set to their Elasticsearch key (`startDate` to `from` and `endDate` to `to`)

Reply:
like TS doesn't like `...(time_zone && { time_zone }),` but it was happy when it was literally just `time_zone` (key and value) 🙄